apps/src/lib/util/audioApi.js

```diff
@@ -15,99 +17,123 @@ export function injectExecuteCmd(fn) {
   executeCmd = fn;
 }
 
-let audioCommands = {};
-
 /**
- *
- * @param {Object} commands Object holding the set of commands for a droplet
- *        toolkit, which is mutated when passed into this function to add
- *        a set of audio commands.
- * @returns {Object} the mutated commands object
+ * Export a set of native code functions that student code can execute via the
+ * interpreter.
+ * Must be mixed in to the app's command list (see applab/commands.js)
  */
-export function addAudioCommands(commands) {
-  for (const commandName in audioCommands) {
-    if (!audioCommands.hasOwnProperty(commandName)) {
-      continue;
-    }
-
-    if (commands.hasOwnProperty(commandName)) {
-      throw new Error(`Attempted to add "${commandName}" to the command set but it already exists`);
-    }
-
-    commands[commandName] = audioCommands[commandName];
-  }
-  return commands;
-}
+export const commands = {
 
-export function playSound(url) {
-  return executeCmd(null, 'playSound', {'url': url});
-}
+  /**
+   * Start playing a sound.
+   * @param {string} opts.url The sound to play.
+   * TODO: Implement additional arguments as part of Sound Library Work
+   *       Spec: https://docs.google.com/document/d/11mpYgmomALyAr53BQl2Ufx0ZYXoMAswqlQBA0aRuNag/edit#heading=h.6uzt0nqaaco
+   * _@param {boolean} [opts.loop] Whether to repeat the sound forever
+   * _@param {boolean} [opts.allowMultiple] If false (default) this call will
+   *        stop other instances of the same sound from playing.  If true,
+   *        multiple instances of the sound may be played simultaneously.
+   */
+  playSound: function playSound(opts) {
```

Comment:
nit, could just be `playSound(opts) {` instead of `playSound: function playSound(opts) {`